tests/testthat/test-derive_var_atoxgr.R

```diff
@@ -9211,3 +10782,264 @@ test_that("derive_var_atoxgr Test 123c: DAIDS White blood cell decreased (USCV u
+    # ANRLO missing - can NOT grade 0 or 1
+    "Acidosis",    7.3,    NA,     7.4,    NA,       9,
+    "Acidosis",    7.34,   NA,     7.4,    NA,       10,
+    "Acidosis",    7.35,   NA,     7.4,    NA,       11,
+    "Acidosis",    7.36,   NA,     7.4,    NA,       12,
+    # AVAL missing cannot grade
+    "Acidosis",    NA,     1.1,    1.4,    NA,       13,
+  ) %>%
+    mutate(AVALU = NA_character_)
+
+  input_acido <- expected_acido %>%
+    select(-ATOXGRL)
+
+  actual_acido <- derive_var_atoxgr_dir(
+    input_acido,
+    new_var = ATOXGRL,
+    meta_criteria = atoxgr_criteria_ctcv6,
+    tox_description_var = ATOXDSCL,
+    criteria_direction = "L",
+    get_unit_expr = AVALU
+  )
+
+  expect_dfs_equal(
+    base = expected_acido,
+    compare = actual_acido,
+    keys = c("TESTNUM")
+  )
+})
+
+
+### Acidosis
+### GRADE_3: pH is >7.5
+### GRADE_1: pH is >normal, but <=7.5
+
+## Test 128: CTCAEv6 Alkalosis (SI unit) ----
+test_that("derive_var_atoxgr Test 128: CTCAEv6 Alkalosis (SI unit)", {
+  expected_alkalo <- tibble::tribble(
+    ~ATOXDSCH,     ~AVAL,  ~ANRLO, ~ANRHI, ~ATOXGRH, ~TESTNUM,
+    "Not a term",  7.3,    7.35,   7.4,    NA,       1,
+    NA_character_, 7.3,    7.35,   7.4,    NA,       2,
+    # ANRHI not missing
+    "Alkalosis",   7.51,   7.35,   7.4,    "3",      3,
+    "Alkalosis",   7.5,    7.35,   7.4,    "1",      4,
+    "Alkalosis",   7.41,   7.35,   7.4,    "1",      5,
+    "Alkalosis",   7.4,    7.35,   7.4,    "0",      6,
+    "Alkalosis",   7.39,   7.35,   7.4,    "0",      7,
+    # ANRHI missing - can grade 3
+    "Alkalosis",   7.51,   7.35,   NA,     "3",      8,
+    # ANRHI missing - can NOT grade 0 or 1
+    "Alkalosis",   7.5,    7.35,   NA,     NA,       9,
+    "Alkalosis",   7.41,   7.35,   NA,     NA,       10,
+    "Alkalosis",   7.4,    7.35,   NA,     NA,       11,
+    "Alkalosis",   7.39,   7.35,   NA,     NA,       12,
+    # AVAL missing cannot grade
+    "Alkalosis",   NA,     1.1,    NA,     NA,       13,
+  ) %>%
+    mutate(AVALU = NA_character_)
+
+  input_alkalo <- expected_alkalo %>%
+    select(-ATOXGRH)
+
+  actual_alkalo <- derive_var_atoxgr_dir(
+    input_alkalo,
+    new_var = ATOXGRH,
+    meta_criteria = atoxgr_criteria_ctcv6,
+    tox_description_var = ATOXDSCH,
+    criteria_direction = "H",
+    get_unit_expr = AVALU
+  )
+
+  expect_dfs_equal(
+    base = expected_alkalo,
+    compare = actual_alkalo,
+    keys = c("TESTNUM")
+  )
+})
+
+## Test when deprecated abnormal_indicator used - should map to high_indicator
+## Test 129: CTCAEv6  Blood bilirubin increased ----
+test_that("derive_var_atoxgr Test 129: CTCAEv6  Blood bilirubin increased", {
+  actual_bili_ctcv6 <- derive_var_atoxgr_dir(
+    input_bili_ctcv6,
+    new_var = ATOXGRH,
+    meta_criteria = atoxgr_criteria_ctcv6,
+    tox_description_var = ATOXDSCH,
+    criteria_direction = "H",
+    abnormal_indicator = "HIGH",
+    get_unit_expr = AVALU
+  )
+
+  expect_snapshot(
```

Comment:
`expect_snapshot()` should be used for the `derive_var_atoxgr_dir()` call as this call issues the message.